src/lib/input/input.scss

```diff
@@ -121,3 +121,21 @@ textarea.mat-input-element {
   padding: 2px 0;
   margin: -2px 0;
 }
+
+// Remove select button from IE11
+select::-ms-expand {
+  display: none;
+}
+
+select[matInput] {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  position: relative;
+  background-color: transparent;
+  background-image: url('data:image/svg+xml;charset=utf8,%3Csvg%20width%3D%2210%22%20height%3D%225%22%20viewBox%3D%227%2010%2010%205%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Cpath%20fill%3D%22%230%22%20fill-rule%3D%22evenodd%22%20opacity%3D%22.54%22%20d%3D%22M7%2010l5%205%205-5z%22%2F%3E%3C%2Fsvg%3E');
```

Comment:
Can you extract the uri portion of this into a sass variable with a comment explaining what it is?

Reply:
AFAIK we can't use data URIs in CSS. See https://github.com/angular/material2/pull/8898.

Reply:
We couldn't think of another way to do this, so we checked directly with Google's security team. They said that they see no reason to ban `data:` URIs, so we can relax this restriction. I do want to add documentation that talks about our CSP support.

Reply:
security team approved using data URI for image.